esp32/README.md

```diff
@@ -91,6 +91,11 @@ $ git submodule init lib/berkeley-db-1.xx
 $ git submodule update
 ```
 
+You will also need to have PySerial installed if it isn't already.
+```bash
+$ pip install pyserial
+```
+
```

Comment:
I think the dependency on pyserial comes from the ESP IDF, since esptool.py uses it.  So at the very least this paragraph should be in the section above about setting up the toolchain, but ideally it would already be part of the ESP IDF documentation (is it?).